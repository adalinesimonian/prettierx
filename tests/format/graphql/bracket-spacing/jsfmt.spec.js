run_spec(__dirname, ["graphql"]);
run_spec(__dirname, ["graphql"], {
  // [prettierx]: broken-out bracket spacing options
  graphqlCurlySpacing: false,
  exportCurlySpacing: false,
  importCurlySpacing: false,
  objectCurlySpacing: false,
  typeCurlySpacing: false,
  yamlBracketSpacing: false,
});
// [prettierx]: broken-out bracket spacing options
run_spec(__dirname, ["graphql"], { graphqlCurlySpacing: false });
