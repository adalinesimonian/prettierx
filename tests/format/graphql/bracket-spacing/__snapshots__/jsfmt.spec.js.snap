// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bracket_spacing.graphql - {"graphqlCurlySpacing":false,"exportCurlySpacing":false,"importCurlySpacing":false,"objectCurlySpacing":false,"typeCurlySpacing":false,"yamlBracketSpacing":false} format 1`] = `
====================================options=====================================
exportCurlySpacing: false
graphqlCurlySpacing: false
importCurlySpacing: false
objectCurlySpacing: false
parsers: ["graphql"]
printWidth: 80
typeCurlySpacing: false
yamlBracketSpacing: false
                                                                                | printWidth
=====================================input======================================
fragment Visit on HighlightedVisit
  @argumentDefinitions(
    count: {type: "Int", defaultValue: 20}
  ) {
  name
}

=====================================output=====================================
fragment Visit on HighlightedVisit
@argumentDefinitions(count: {type: "Int", defaultValue: 20}) {
  name
}

================================================================================
`;

exports[`bracket_spacing.graphql - {"graphqlCurlySpacing":false} format 1`] = `
====================================options=====================================
graphqlCurlySpacing: false
parsers: ["graphql"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
fragment Visit on HighlightedVisit
  @argumentDefinitions(
    count: {type: "Int", defaultValue: 20}
  ) {
  name
}

=====================================output=====================================
fragment Visit on HighlightedVisit
@argumentDefinitions(count: {type: "Int", defaultValue: 20}) {
  name
}

================================================================================
`;

exports[`bracket_spacing.graphql format 1`] = `
====================================options=====================================
parsers: ["graphql"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
fragment Visit on HighlightedVisit
  @argumentDefinitions(
    count: {type: "Int", defaultValue: 20}
  ) {
  name
}

=====================================output=====================================
fragment Visit on HighlightedVisit
@argumentDefinitions(count: { type: "Int", defaultValue: 20 }) {
  name
}

================================================================================
`;
